docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md

```diff
@@ -315,6 +400,14 @@ The following represents a collection of relevant documentation regarding the pr
                   }
               ]
           }
+      },
+      {
+        "key" : "assetLifecyclePhase",
+        "value": "AsBuilt"
+      },
```

Comment:
This is not mandated by the industry core for R24-05 anymore. Leave it in only if you want to go beyond the compatibility guarantees of CX.

Reply:
@matbmoser should we leave this is ?

Reply:
You can remove it, we need just the digital twin type, that's mandatory